lib/resty/upload.lua

```diff
@@ -5,7 +5,6 @@ local sub = string.sub
 local req_socket = ngx.req.socket
 local match = string.match
 local setmetatable = setmetatable
-local error = error
```

Comment:
This is important since we would never read a global variable named `error` in per-request code paths, which is kinda expensive.